examples/UnicycleTrajectoryGenerator/main.cpp

```diff
@@ -205,23 +206,24 @@ int main(int argc, char* argv[])
         w_H_left = leftFootPlanner.getOutput().transform;
         w_H_right = rightFootPlanner.getOutput().transform;
 
+        // get the DCM trajectory from the unicycle trajectory generator
+        auto dcmPosition = output.dcmTrajectory.position;
+        auto dcmVelocity = output.dcmTrajectory.velocity;
```

Comment:
I think this should avoid new memory allocation 
```suggestion
        const auto& dcmPosition = output.dcmTrajectory.position;
        const auto& dcmVelocity = output.dcmTrajectory.velocity;
```